Assets/SEE/Utils/Markdown/RichTagsMarkdownBlockRenderers.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using Markdig.Syntax;
+
+namespace SEE.Utils.Markdown
+{
+    public partial class RichTagsMarkdownRenderer
```

Comment:
Dito. Would be good to have a description of the particular purpose of this part.

Reply:
Fixed.